### PR TITLE
test cases for reset_secret

### PR DESCRIPTION
**Pull request by shreeyan08**
Please review "method reset_secret" and "testcases for reset_secret method"

### CODE REVIEW

File: pythonAPIClient/client.py [hunk 1]
```diff
@@ -229,6 +229,41 @@ def update_permission(self, clientID, permissionLevel):
...
                 raise InternalError()
         return response.json()['message']
+	
+    #TODO : reset Secret
+    def reset_secret(self, clientID):
+        """
+        Parameters
+        ----------
+
+        cliendID: str
```
Comment (mush-zhang) [inline pythonAPIClient/client.py]: *clientID* typo

**Review by mush-zhang — CHANGES_REQUESTED** [review]

**Review by jaydenC88 — APPROVED** [review]
Good work!

**Review by moealani — APPROVED** [review]
Nice work just need to fix the typos

**Review by Mrjimjr — APPROVED** [review]
Good Work

**Review by mush-zhang — APPROVED** [review]

**Comment by mush-zhang** [comment]
Shreeyan created a feature branch of the identical change. we shall close this pr generated from fork.

**Pull request closed without merging**